.github/workflows/sdk_protos_map.csv

```diff
@@ -1,480 +1,478 @@
-board,GetResourceName,No,get_resource_name,,getResourceName
-board,Close,No,close,Close,
+board,GetGeometries,No,get_geometries,,,
+board,Reconfigure,No,,Reconfigure,,
+board,DoCommand,Yes,do_command,DoCommand,doCommand,doCommand
+board,Name,No,,Name,,
+board,GetResourceName,No,get_resource_name,,getResourceName,
+board,Close,No,close,Close,,
 
 ## Button
-button,Push,No,,Push,
-button,DoCommand,No,,DoCommand,
-button,Close,No,,Close,
+button,Push,No,,Push,,push
+button,DoCommand,No,,DoCommand,,doCommand
+button,Close,No,,Close,,
 
 ## Camera
-camera,GetImage,,get_image,Image,image
-camera,GetImages,,get_images,Images,
-camera,RenderFrame,,,,
-camera,GetPointCloud,,get_point_cloud,NextPointCloud,pointCloud
-camera,GetProperties,,get_properties,Properties,properties
+camera,GetImage,,get_image,Image,image,getImage
+camera,GetImages,,get_images,Images,,
+camera,RenderFrame,,,,,renderFrame
+camera,GetPointCloud,,get_point_cloud,NextPointCloud,pointCloud,getPointCloud
+camera,GetProperties,,get_properties,Properties,properties,getProperties
 ## TED: Camera in Go SDK doesn't appear to implement (inherit) these:
-camera,DoCommand,,do_command,,doCommand
-camera,GetGeometries,,get_geometries,,
+camera,DoCommand,,do_command,,doCommand,doCommand
+camera,GetGeometries,,get_geometries,,,getGeometries
 ## HACK:  proto for close, manually mapping:
-camera,GetResourceName,,get_resource_name,,getResourceName
-camera,Close,,close,Close,
+camera,GetResourceName,,get_resource_name,,getResourceName,
+camera,Close,,close,Close,,
 
 ## Encoder
-encoder,GetPosition,Yes,get_position,Position,
-encoder,ResetPosition,Yes,reset_position,ResetPosition,
-encoder,GetProperties,Yes,get_properties,Properties,
-encoder,GetGeometries,No,get_geometries,,
+encoder,GetPosition,Yes,get_position,Position,,getPosition
+encoder,ResetPosition,Yes,reset_position,ResetPosition,,resetPosition
+encoder,GetProperties,Yes,get_properties,Properties,,getProperties
+encoder,GetGeometries,No,get_geometries,,,
 ## HACK: No proto for these (and/or inherited in Go SDK), manually mapping:
-encoder,Reconfigure,No,,Reconfigure,
-encoder,DoCommand,Yes,do_command,DoCommand,
-encoder,GetResourceName,No,get_resource_name,,
-encoder,Close,No,close,Close,
+encoder,Reconfigure,No,,Reconfigure,,
+encoder,DoCommand,Yes,do_command,DoCommand,,doCommand
+encoder,GetResourceName,No,get_resource_name,,,
+encoder,Close,No,close,Close,,
 
 ## Gantry
-gantry,GetPosition,,get_position,Position,position
-gantry,MoveToPosition,,move_to_position,MoveToPosition,moveToPosition
-gantry,GetLengths,,get_lengths,Lengths,lengths
-gantry,Home,,home,Home,home
+gantry,GetPosition,,get_position,Position,position,getPosition
+gantry,MoveToPosition,,move_to_position,MoveToPosition,moveToPosition,moveToPosition
+gantry,GetLengths,,get_lengths,Lengths,lengths,getLengths
+gantry,Home,,home,Home,home,home
 ## TED: Gantry in Go SDK doesn't appear to implement (inherit) this:
-gantry,GetGeometries,,,,
+gantry,GetGeometries,,,,,getGeometries
```

Comment:
looks like Python get_geometries was removed from the duplicate line below but should be here

Reply:
yep, fixing in https://github.com/viamrobotics/docs/pull/4090